packages/generator-electrode/generators/app/templates/_package.json

```diff
@@ -42,6 +42,7 @@
     "koa-router": "^5.4.0",
     "koa-send": "^3.2.0",
     "koa-static": "^2.0.0", <% } if (isPWA) { %>
+    "react": "^16.3.2",
```

Comment:
is the archetype not working w/o this?

Reply:
same here, in my tests it didn't work, due to `react-notify-toast` dependency